openduck-py/openduck_py/settings.py

```diff
@@ -0,0 +1 @@
+IS_DEV = True
```

Comment:
This should be in https://github.com/uberduck-ai/openduck/blob/main/openduck-py/openduck_py/settings/__init__.py and you'll want to set `IS_DEV` in your .env file

Reply:
I see @zachwe beat me to the punch

Reply:
race condition